Podfile

```diff
@@ -18,7 +18,7 @@ target 'Sequel Ace' do
   pod 'SwiftLint', '~> 0.40'
   pod 'FirebaseCore'
   pod 'Firebase/Crashlytics'
-  pod 'FMDB', :git => 'https://github.com/ccgus/fmdb.git', :tag => '2.7.7' #2.7.7 is not published to the specs repo yet, only 2.7.5, so targeting the tag.
+  pod 'FMDB', :git => 'https://github.com/jamesstout/fmdb.git', :branch => 'update_podspec' #2.7.7 is not published to the specs repo yet, only 2.7.5, so targeting the tag.
```

Comment:
Would it be worth moving this repo into the sequelace GitHub team?

Reply:
You are looking at outdated diff :)